[Release/1.0.0]System.Net.Security.Tests failing on Opensuse

```
15:50:21   Copyright (C) 2014 Outercurve Foundation.
15:50:21   
15:50:21   Discovering: System.Net.Security.Unit.Tests
15:50:21   Discovered:  System.Net.Security.Unit.Tests
15:50:21   Starting:    System.Net.Security.Unit.Tests
15:50:22   Finished:    System.Net.Security.Tests
15:50:22   
15:50:22   === TEST EXECUTION SUMMARY ===
15:50:22      System.Net.Security.Tests  Total: 97, Errors: 0, Failed: 0, Skipped: 3, Time: 3.204s
15:50:22   Finished running tests. End time=22:51:38. Return value was 0
15:50:22 /mnt/resource/j/workspace/dotnet_corefx/release_1.0.0/opensuse13.2_debug/Tools/tests.targets(202,5): warning MSB3073: The command "/mnt/resource/j/workspace/dotnet_corefx/release_1.0.0/opensuse13.2_debug/bin/tests/Linux.AnyCPU.Debug/System.Net.Security.Tests/netcoreapp1.0/RunTests.sh /mnt/resource/j/workspace/dotnet_corefx/release_1.0.0/opensuse13.2_debug/packages/" exited with code -1. [/mnt/resource/j/workspace/dotnet_corefx/release_1.0.0/opensuse13.2_debug/src/System.Net.Security/tests/FunctionalTests/System.Net.Security.Tests.csproj]
15:50:22 /mnt/resource/j/workspace/dotnet_corefx/release_1.0.0/opensuse13.2_debug/Tools/tests.targets(211,5): error : One or more tests failed while running tests from 'System.Net.Security.Tests' please check log for details! 
```

--- a/src/System.Net.Security/tests/FunctionalTests/TestConfiguration.cs
+++ b/src/System.Net.Security/tests/FunctionalTests/TestConfiguration.cs
@@ -40,8 +40,14 @@ namespace System.Net.Security.Tests
             // On Unix, it depends on how openssl was built.  So we ask openssl if it has any.
             try
             {
-                using (Process p = Process.Start(new ProcessStartInfo("openssl", "ciphers NULL") { RedirectStandardOutput = true }))
+                using (Process p = Process.Start(new ProcessStartInfo("openssl", "ciphers NULL") { RedirectStandardOutput = true, RedirectStandardError = true }))
                 {
+                    // On some platforms (openSUSE 13.2 is one example), doing this query can print error messages to standard error
+                    // when the tests are run via MSBuild, this error message gets picked up and treated as an error from the test itself
+                    // causing the task to fail.  We don't actually care about the error text at all, so we just ignore it.
+                    p.ErrorDataReceived += ((object sendingProcess, DataReceivedEventArgs errorText) => { /* ignore */ });
+                    p.BeginErrorReadLine();
+
                     return p.StandardOutput.ReadToEnd().Trim().Length > 0;
                 }
             }

